contracts/multisig-prover/src/encoding/sol/mod.rs

```diff
@@ -0,0 +1,116 @@
+use std::collections::BTreeMap;
+
+use axelar_encoding::types::{
+    CrossChainId, Message, Payload, PublicKey, Signature, Signer, WeightedSignature, WorkerSet,
+    U256,
+};
+
+use itertools::Itertools;
+use multisig::{msg::SignerWithSig, verifier_set::VerifierSet};
+
+const ECDSA_COMPRESSED_PUBKEY_LEN: usize = 33; // this should be probably a public constant in axelar_encoding.
+const ED25519_PUBKEY_LEN: usize = 32; // this should be probably a public constant in axelar_encoding.
+
```

Comment:
This 2 constants are private in the `axelar-encoding` repo. Maybe we could make it public.

Reply:
Thanks ! implemented at 23be5851636afac76128f097f0353aa15bc3b7fe